maro/simulator/scenarios/citi_bike/station.py

```diff
@@ -48,6 +51,7 @@ def reset(self):
         self.capacity = self._init_capacity
         self.bikes = self._init_bikes
         self.min_bikes = self._init_bikes
+        self.id = self._id
 
```

Comment:
Do we need rm weather in the station?

Reply:
@chaosddp

Reply:
at least remove weather reader from be,  or not exist file path may cause error